README.md

```diff
@@ -142,11 +142,16 @@ After deleting your cluster deployment you will see an uninstall job created. If
 
  1. Delete the uninstall job, it will be recreated and tried again.
  1. Manually delete the uninstall finalizer allowing the cluster deployment to be deleted, but note that this may leave artifacts in your AWS account.
-    * `kubectl edit clusterdeployments.hive.openshift.io YOURCLUSTER`
- 1. You can manually run the uninstall code with hiveutil to delete AWS resources based on their tags.
-    * Get your cluster UUID from the clusterdeployment.Spec.ClusterUUID.
-    * `make hiveutil`
-    * `bin/hiveutil aws-tag-deprovision --loglevel=debug --cluster-name CLUSTER_NAME openshiftClusterID=CLUSTER_UUID kubernetes.io/cluster/CLUSTER_NAME=owned`
+ 1. You can manually run the uninstall code with `hiveutil` to delete AWS resources based on their tags.
+    * Run `make hiveutil`
+    * Get your cluster tag from the following command output. Look for infraID in status.infraID
```

Comment:
This should clarify that if your cluster deployment is gone you can also find the tag in AWS console on almost any cloud object from that cluster.

Reply:
Agree